packages/components/src/DataList/components/DataListItem/DataListItem.test.tsx

```diff
@@ -106,7 +116,26 @@ describe("DataListItem", () => {
 
       const listItemEl = screen.getByText(listItem);
       await userEvent.hover(listItemEl);
-      fireEvent.contextMenu(listItemEl);
+      await userEvent.pointer({ keys: "[MouseRight>]", target: listItemEl });
+
+      expect(screen.queryByRole("menu")).not.toBeInTheDocument();
+      expect(
+        screen.queryByRole("button", { name: "More actions" }),
+      ).not.toBeInTheDocument();
+    });
+
+    it("should not show a context menu when at least one item is selected", async () => {
+      mockItemActionComponent.mockReturnValueOnce(
+        <DataListItemActions onClick={handleItemClick}>
+          <DataListAction label="Edit" visible={() => false} />
```

Comment:
The menu would never show up even if selected because the actions aren't visible. 
```suggestion
          <DataListAction label="Edit" />
```